styles/helpers/bg-bleed.css

```diff
@@ -0,0 +1,9 @@
+.bg-bleed {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1600px;
+  background-size: cover;
```

Comment:
I feel like this shouldn't be added or assumed because its not always a background image.  We should pass this in maybe as a part of a mixin.